packages/components/popover/src/popover-trigger.tsx

```diff
@@ -38,11 +38,22 @@ const PopoverTrigger = forwardRef<"button", PopoverTriggerProps>((props, _) => {
 
   const {buttonProps} = useAriaButton({onPress}, triggerRef);
 
+  let restProps = rest;
+
   const hasNextUIButton = useMemo<boolean>(() => {
     return triggerChildren?.[0] !== undefined;
   }, [triggerChildren]);
 
-  return cloneElement(child, mergeProps(rest, hasNextUIButton ? {onPress} : buttonProps));
+  if (restProps?.isDisabled) {
+    // if `child` doesn't have `isDisabled` prop, e.g. custom trigger (div), NextUI User component
+    // adding `isDisabled` would make React fail to recognize it on a DOM element
+    // hence, adding the `isDisabled` logic to cover this case
+    restProps.className = cn(restProps.className, "opacity-disabled pointer-events-none");
```

Comment:
@wingkwong we can't use tailwindcss classes outside the `theme` package, this is because users load the tailwindcss styles only from the `theme` package https://nextui.org/docs/guide/installation#tailwind-css-setup

I think we could manage this scenario directly in the `getTriggerProps`